### PR TITLE
turn on stats collect by default

### PR DESCRIPTION
**Pull request by max-hoffman**

### CODE REVIEW

**Comment by max-hoffman** [comment]
#benchmark

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>26010b7</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>26010b7</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 |      read_tests      | from_latency | to_latency | percent_change |
|----------------------|--------------|------------|----------------|
| covering_index_scan  | 0.63         | 0.63       | 0.0            |
| groupby_scan         | 16.71        | 16.71      | 0.0            |
| index_join           | 2.3          | 2.26       | -1.74          |
| index_join_scan      | 1.89         | 1.89       | 0.0            |
| index_scan           | 55.82        | 55.82      | 0.0            |
| oltp_point_select    | 0.27         | 0.27       | 0.0            |
| oltp_read_only       | 5.47         | 5.47       | 0.0            |
| select_random_points | 0.61         | 0.61       | 0.0            |
| select_random_ranges | 0.62         | 0.62       | 0.0            |
| table_scan           | 56.84        | 56.84      | 0.0            |
| types_table_scan     | 147.61       | 147.61     | 0.0            |


|      write_tests      | from_latency | to_latency | percent_change |
|-----------------------|--------------|------------|----------------|
| oltp_delete_insert    | 5.88         | 5.88       | 0.0            |
| oltp_insert           | 2.91         | 2.91       | 0.0            |
| oltp_read_write       | 11.24        | 11.24      | 0.0            |
| oltp_update_index     | 2.97         | 2.97       | 0.0            |
| oltp_update_non_index | 2.91         | 2.91       | 0.0            |
| oltp_write_only       | 5.99         | 5.99       | 0.0            |
| types_delete_insert   | 6.43         | 6.43       | 0.0            |

**Comment by max-hoffman** [comment]
#benchmark

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>a25076d</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>a25076d</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 |      test_name      | from_latency_p95 | to_latency_p95 | percent_change |
|---------------------|------------------|----------------|----------------|
| tpcc-scale-factor-1 | 65.65            | 63.32          | -3.55          |


|      test_name      | from_server_name |           from_server_version            | from_tps | to_server_name |            to_server_version             | to_tps | percent_change |
|---------------------|------------------|------------------------------------------|----------|----------------|------------------------------------------|--------|----------------|
| tpcc-scale-factor-1 | dolt             | 73df10eef0ab98e327dcaafda361103d65105371 | 40.12    | dolt           | a25076debd09f39ec98f101314d67334a82f09b6 | 40.14  | 0.05           |

**Comment by coffeegoddd** [comment]
@coffeegoddd __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>c07b2d7</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>c07b2d7</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 |      read_tests      | from_latency | to_latency | percent_change |
|----------------------|--------------|------------|----------------|
| covering_index_scan  | 0.62         | 0.62       | 0.0            |
| groupby_scan         | 16.71        | 16.71      | 0.0            |
| index_join           | 2.3          | 2.26       | -1.74          |
| index_join_scan      | 1.86         | 1.89       | 1.61           |
| index_scan           | 55.82        | 55.82      | 0.0            |
| oltp_point_select    | 0.27         | 0.27       | 0.0            |
| oltp_read_only       | 5.47         | 5.47       | 0.0            |
| select_random_points | 0.61         | 0.61       | 0.0            |
| select_random_ranges | 0.63         | 0.62       | -1.59          |
| table_scan           | 55.82        | 56.84      | 1.83           |
| types_table_scan     | 144.97       | 144.97     | 0.0            |


|      write_tests      | from_latency | to_latency | percent_change |
|-----------------------|--------------|------------|----------------|
| oltp_delete_insert    | 5.88         | 5.88       | 0.0            |
| oltp_insert           | 2.91         | 2.91       | 0.0            |
| oltp_read_write       | 11.24        | 11.24      | 0.0            |
| oltp_update_index     | 2.97         | 2.97       | 0.0            |
| oltp_update_non_index | 2.91         | 2.91       | 0.0            |
| oltp_write_only       | 5.99         | 5.99       | 0.0            |
| types_delete_insert   | 6.43         | 6.43       | 0.0            |

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>e97d407</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>e97d407</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@coffeegoddd __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>133f228</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>133f228</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>d398a43</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>d398a43</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>9fb6704</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>9fb6704</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

File: go/cmd/dolt/commands/sqlserver/server.go [hunk 1]
```diff
@@ -271,8 +271,11 @@ func ConfigureServices(
 	var sqlEngine *engine.SqlEngine
 	InitSqlEngine := &svcs.AnonService{
 		InitF: func(ctx context.Context) (err error) {
-			if _, err := mrEnv.Config().GetString(env.SqlServerGlobalsPrefix + "." + dsess.DoltStatsBootstrapEnabled); err != nil {
-				// unless otherwise specified by config, enable server bootstrapping
+			if _, err := mrEnv.Config().GetString(env.SqlServerGlobalsPrefix + "." + dsess.DoltStatsAutoRefreshEnabled); err != nil {
```
Comment (zachmu) [inline go/cmd/dolt/commands/sqlserver/server.go:274]: This logic is pretty weird. Variable is named "enabled" but we're treating it in the logic as "not disabled". Which is it? What is the default? Can it be set to "false"?

**Review by zachmu — APPROVED** [review]
LGTM

File: go/cmd/dolt/commands/sqlserver/server.go (hunk 1 above)
Comment (max-hoffman) [reply go/cmd/dolt/commands/sqlserver/server.go:274]: ok, so just invert the naming now that we've inverted default behavior?

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>a6d5608</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>a6d5608</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

File: go/cmd/dolt/commands/sqlserver/server.go (hunk 1 above)
Comment (max-hoffman) [reply go/cmd/dolt/commands/sqlserver/server.go:274]: On a second look, i think the confusing part is that we previously decided `sql-server` is the only context where we enable default stats behavior. So it is default off for everything but `sql-server`, and default on for `sql-server` unless the config explicitly specifies a behavior. I'm going to just document better and still ship the current behavior/naming.

**Comment by coffeegoddd** [comment]
@max-hoffman __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>a020d8c</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>a020d8c</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Pull request merged**